Dockerfile

```diff
@@ -14,6 +14,7 @@ ENV HOME=/opt/pelias
 
 #Set geotrans IP
 ENV GEOTRANS_IP=
```

Comment:
Lets just mae a GEOTRANS_URL parameter, then it will also support a difference in the protocol (i.e. https), as well as having less paramters to configure.